whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -257,7 +261,9 @@ private RdapObject getRdapObject(final String requestUrl,
     private RdapObject mapCommons(final RdapObject rdapResponse, final String requestUrl) {
         rdapResponse.getNotices().add(noticeFactory.generateTnC(requestUrl));
 
-        rdapResponse.getRdapConformance().addAll(RDAP_CONFORMANCE_LEVEL);
+        rdapResponse.getRdapConformance().addAll(List.of(RdapConformance.CIDR_0.getValue(),
+                RdapConformance.LEVEL_0.getValue(),
```

Comment:
I think cidr0 is only for inetnum and inet6num, this will add  it to every object type? Can you confirm ?

Reply:
I just read your reply to Ed, seems like it should be included in every response